drake/bindings/python/pydrake/test/testRBTCoM.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import print_function
```

Comment:
So we're doing python 2? I hear that people are finally switching to python 3; should we just do python 3 from the start?